base/dcp_dest.go

```diff
@@ -80,7 +82,8 @@ func NewDCPDest(ctx context.Context, callback sgbucket.FeedEventCallbackFunc, bu
 		DCPCommon:          dcpCommon,
 		stats:              dcpStats,
 		partitionCountStat: importPartitionStat,
-		metaInitComplete:   make([]bool, maxVbNo),
+		metaInitComplete:   make([]bool, dcpCommon.maxVbNo),
```

Comment:
These values should be the same, but let's keep as it was:

```suggestion
		metaInitComplete:   make([]bool, maxVbNo),
```